### PR TITLE
Updated flutter_localizations

### PR DESCRIPTION
**Pull request by HansMuller**
Updated translations for all locales. 

Added country-specific translations:
gsw (Swiss German)
de_CH (German - Switzerland)
en_AU (English - Australia)
en_CA (English - Canada)
en_IN (English - India)
en_SG (English - Singapore)
pt_PT (Portugese - Portugal)

### CODE REVIEW

File: packages/flutter_localizations/lib/src/l10n/localizations.dart [hunk 1]
```diff
@@ -12,31 +12,36 @@
 /// This variable is used by [MaterialLocalizations].
 const Map<String, Map<String, String>> localizations = const <String, Map<String, String>> {
```
Comment (Hixie) [inline packages/flutter_localizations/lib/src/l10n/localizations.dart]: we really should move away from this map to just hard-coded classes

File: packages/flutter_localizations/lib/src/l10n/localizations.dart [hunk 2]
```diff
@@ -57,17 +62,49 @@ const Map<String, Map<String, String>> localizations = const <String, Map<String
...
+    'continueButtonLabel': r'WEITER',
     'copyButtonLabel': r'KOPIEREN',
     'cutButtonLabel': r'AUSSCHNEIDEN',
     'okButtonLabel': r'OK',
     'pasteButtonLabel': r'EINFÜGEN',
-    'selectAllButtonLabel': r'ALLES AUSWÄHLEN',
+    'selectAllButtonLabel': r'ALLE AUSWÄHLEN',
     'viewLicensesButtonLabel': r'LIZENZEN ANZEIGEN',
+    'anteMeridiemAbbreviation': r'VORM.',
+    'postMeridiemAbbreviation': r'NACHM.',
```
Comment (Hixie) [inline packages/flutter_localizations/lib/src/l10n/localizations.dart]: cc @goderbauer for review

**Comment by Hixie** [comment]
RSLGTM

File: packages/flutter_localizations/lib/src/l10n/localizations.dart (hunk 2 above)
Comment (goderbauer) [reply packages/flutter_localizations/lib/src/l10n/localizations.dart]: LGTM

**Pull request merged**